chain-abci/src/staking/tx.rs

```diff
@@ -24,9 +24,9 @@ impl StakingTable {
         heap: &mut impl StoreStaking,
         block_time: Timespec,
         max_evidence_age: Timespec,
-        ra_verifier: &EnclaveCertVerifier,
+        mut recent_isv_svn: u16,
         tx: &NodeJoinRequestTx,
-    ) -> Result<(), PublicTxError> {
+    ) -> Result<u16, PublicTxError> {
```

Comment:
why it needs it `mut` when it also returns it?

Reply:
It'll get assigned if `info.quote.report_body.isv_svn > recent_isv_svn`, and returned at the end.

Reply:
how about 
`recent_isv_svn:&mut u16` ?
for readibility

Reply:
Used a new variable to return now, it would be less confusing.